recipes/openlineage-dbt/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "openlineage-dbt" %}
+{% set version = "0.30.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openlineage-dbt-{{ version }}.tar.gz
+  sha256: 6341131b97369c01baed5c8ccb5654e1f9ae73832bbe89046c204beb25596e66
+
+build:
+  skip: true  # [py<38]
```

Comment:
This one has scripts so it needs to be arch'ed:

```
scripts=['scripts/dbt-ol']
```

Reply:
Or use https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies